src/main/java/com/exactpro/th2/crawler/Crawler.java

```diff
@@ -503,6 +532,8 @@ private void putAndCheck(Map<StreamKey, InnerMessageId> checkpoints, Map<StreamK
             MessageID messageId = data.getMessageId();
             MessageID checkpointId = checkpointByDirection.get(createStreamKeyFrom(messageId));
             String sessionAlias = messageId.getConnectionId().getSessionAlias();
+            // Update the last message for alias + direction
+            metrics.lastMessage(sessionAlias, messageId.getDirection(), data);
```

Comment:
Could we move this code close to the updateRecoveryState call to reflect the actual state of database